docs/netdata-agent/configuration/dynamic-configuration.md

```diff
@@ -0,0 +1,63 @@
+# Dynamic Configuration Manager
+
+**Netdata Cloud paid subscriptions required.**
+
+The Dynamic Configuration Manager allows direct configuration of collectors and alerts through the Netdata UI. This
+feature allows users to:
+
+- **Create, test, and deploy configurations** for one or more nodes directly within the UI.
+- **Eliminate the need for manual command-line edits and node access**, enhancing workflow efficiency.
+
+**Cloud Connection and Security**: Nodes using Dynamic Configuration Manager require a connection to Netdata Cloud. This
+ensures proper permission handling and data security.
+
+**Permissions**: Admins and Managers can view, edit, and submit configurations within the UI.
```

Comment:
@Ancairon we need to improve this. We need to add a table to https://learn.netdata.cloud/docs/netdata-cloud/authentication-&-authorization/role-based-access-model and add a link to it here

Reply:
correct!